src/api/services/enrichment/enrichment.js

```diff
@@ -77,13 +81,14 @@ export const getEnrichmentDataPreview = async ctx => {
 };
 
 const isBasicType = sourceData => {
-    return typeof sourceData === 'string' || typeof sourceData === 'number';
+    return typeof sourceData !== 'object';
```

Comment:
What about date types? Do we actually handle them? typeof a date is 'object' too

Reply:
Date are serialized as String, so I think it's good.